packages/components/src/internal/components/chart/BarChartViewer.tsx

```diff
@@ -250,21 +251,34 @@ export class BarChartViewer extends PureComponent<Props, State> {
                         </Tip>
                     </div>
                 )}
-                {!hasError && hasSectionItems && !!selectedGroup.createURL && !!selectedGroup.createText && (
-                    <RequiresPermission perms={PermissionTypes.Insert}>
-                        <div className="pull-right">
-                            <Button
-                                bsStyle="primary"
-                                className="button-right-spacing"
-                                href={selectedGroup.createURL().toHref()}
-                            >
-                                {selectedGroup.createText}
-                            </Button>
-                        </div>
-                    </RequiresPermission>
-                )}
+                {!hasError && hasSectionItems && selectedGroup.showSampleButtons && <SampleButtons />}
                 <div className="margin-top">{body}</div>
             </Section>
         );
     }
 }
+
+const SampleButtons: FC = () => {
+    return (
+        <RequiresPermission perms={PermissionTypes.Insert}>
```

Comment:
You shouldn't need Insert permissions to see Sample Finder.

Reply:
Fixed